arviz/plots/posteriorplot.py

```diff
@@ -358,6 +358,7 @@ def format_axes():
             fill_kwargs={"alpha": kwargs.pop("fill_alpha", 0)},
             plot_kwargs={"linewidth": linewidth},
             ax=ax,
+            rug=True,
```

Comment:
The default value should be `False`, just to keep this plot close to the originals by Kruschke